apps/web/src/utils/utils.ts

```diff
@@ -1,3 +1,24 @@
 export function isBrowser() {
   return typeof window !== 'undefined';
 }
+
+export function formatNumber(num, digits) {
```

Comment:
```suggestion
export function formatNumber(num: number, digits: number): string {
```

Worth adding a TS interface for it

Reply:
Sure thing.